tests/toranj/openthread-core-toranj-config.h

```diff
@@ -190,8 +190,6 @@
 
 #define OPENTHREAD_CONFIG_DELAY_AWARE_QUEUE_MANAGEMENT_ENABLE 1
 
-#define OPENTHREAD_CONFIG_CLI_REGISTER_IP6_RECV_CALLBACK 1
-
```

Comment:
Let's keep it with a #ifndef guard:
```
#ifndef OPENTHREAD_CONFIG_CLI_REGISTER_IP6_RECV_CALLBACK
#define OPENTHREAD_CONFIG_CLI_REGISTER_IP6_RECV_CALLBACK 1
#endif
```
Thus we dont' need to pass the option in CMake in most cases.